pkg/executor/slow_query_test.go

```diff
@@ -550,6 +550,26 @@ func TestSplitbyColon(t *testing.T) {
 			[]string{"1a", "4d"},
 			[]string{"[2b,3c]", "5e"},
 		},
+		{
+			"1a: [2b,[3c: 3cc]] 4d: 5e",
+			[]string{"1a", "4d"},
+			[]string{"[2b,[3c: 3cc]]", "5e"},
+		},
+		{
+			"1a: {2b 3c} 4d: 5e",
+			[]string{"1a", "4d"},
+			[]string{"{2b 3c}", "5e"},
+		},
+		{
+			"1a: {2b,3c} 4d: 5e",
+			[]string{"1a", "4d"},
+			[]string{"{2b,3c}", "5e"},
+		},
+		{
+			"1a: {2b,{3c: 3cc}} 4d: 5e",
+			[]string{"1a", "4d"},
+			[]string{"{2b,{3c: 3cc}}", "5e"},
+		},
```

Comment:
- Add some invalid cases?
- Maybe we can add some cases that multi left brackets stay together such as "{{{" or "[[["

Reply:
Good suggestion, will add it.

Reply:
Done